src/extrapolation/filled.jl

```diff
@@ -54,6 +54,16 @@ end
     return ret
 end
 
+@inline function Interpolations.gradient(etp::FilledExtrapolation{T}, x::Vararg{Number,N}) where {T,N}
+    if checkbounds(Bool, etp.itp, x...)
+        gradient(etp.itp, x...)
```

Comment:
`parent(etp)` instead of `etp.itp` to be symmetric with the other `gradient` function?

Reply:
Okay that sounds fine.